docs/api/backend.md

```diff
@@ -0,0 +1,596 @@
+---
+title: Backend
+layout: default
+parent: API
+---
+
+# Backend
+{: .no_toc }
+
+1. TOC
+{:toc}
+
+The `Backend` class represents the server-side instance of ShareDB. It is primarily responsible for connecting to clients, and sending requests to the [database adapters]({% link adapters/database.md %}).
+
+It is also responsible for some [configuration](#backend-constructor), setting up [middleware]({% link middleware/index.md %}) and defining [projections]({% link projections.md %}).
+
+## Backend() constructor
+
+```js
+var Backend = require('sharedb')
+new Backend([options])
+```
+
+### Options
+
+{: .d-inline-block }
+
+`db` -- [DB]({% link adapters/database.md %})
+
+Optional
+{: .label .label-grey }
+
+> Default: new `MemoryDB` instance
+
+> An instance of a ShareDB [database adapter]({% link adapters/database.md %}) that provides the data store for ShareDB
+
+{: .warn }
+> The default option -- a new `MemoryDB` instance -- is a **non-persistent**, in-memory adapter and should **not** be used in production environments.
+
+{: .d-inline-block }
+
+`pubsub` -- [PubSub]({% link adapters/pub-sub.md %})
+
+Optional
+{: .label .label-grey }
+
+> Default: new `MemoryPubSub` instance
+
+> An instance of a ShareDB [Pub/Sub adapter]({% link adapters/pub-sub.md %}) that provides a channel for notifying other ShareDB instances of changes to data.
+
+{: .info }
+> The default option -- a new `MemoryPubSub` instance -- is an in-memory adapter.
+>
+> Unlike the [database adapter](#db--db), the in-memory Pub/Sub adapter *may* be used in a production environment, where Pub/Sub state need only persist across a single, stand-alone server.
+
+{: .d-inline-block }
+
+`milestoneDb` -- [MilestoneDB]({% link adapters/milestone.md %})
+
+Optional
+{: .label .label-grey }
+
+> Default: `null`
+
+> An instance of a ShareDB [milestone adapter]({% link adapters/milestone.md %}) that provides the data store for [milestone snapshots]({% link document-history.md %}#milestone-snapshots), which are historical snapshots of documents stored at a specified version interval.
+
+{: .info }
+> If this option is omitted, milestone snapshots will **not** be enabled, but historic snapshots *may* still be fetched.
```

Comment:
nit: Can we explain a bit more? I don't really understand how the two are different.